internal/session/repository_connection.go

```diff
@@ -399,12 +399,13 @@ func (r *ConnectionRepository) closeOrphanedConnections(ctx context.Context, wor
 		notInClause = fmt.Sprintf(notInClause, strings.Join(params, ","))
 	}
 
+	query := fmt.Sprintf(orphanedConnectionsCte, notInClause)
```

Comment:
Unsolicited nit: should we rename this to something like closeOrphanedConnections since this does not appear to be a cte anymore

Reply:
Fixed